File: Assignment___39/Assignment_1.cpp
#include<iostream>
using namespace std;

template <class T>
void Display(T Value, int iSize)
{
    for(int iCnt = 1; iCnt <= iSize; iCnt++)
    {
        cout << Value << " ";
    }
    cout << "\n";
}

int main()
{
    Display<int>(11, 5);      
    Display<float>(3.14f, 3); 
    Display<char>('A', 4); 

    return 0;
}

File: Assignment___52/Assignment_1.java
import java.util.*;



class MyArray
{
    private int Arr[];

    public MyArray(int iSize)
    {
        Arr = new int[iSize];
    }

    public void Accept()
    {
        Scanner sobj = new Scanner (System.in);
        System.out.println("Enter "+ Arr.length+" Elements : ");

        for(int i = 0 ; i < Arr.length; i++)
        {
            Arr[i] = sobj.nextInt();
        }
    }

    public void Display()
    {
        for(int i = 0 ; i < Arr.length; i++)
        {
            System.out.print(Arr[i]+"\t");
        }
        System.out.println();
    }


}
class Assignment_1
{

    public static void main(String A[])
    {
        Scanner sobj = new Scanner (System.in);

        System.out.println("Enter Size of first Array : ");
        int iValue1 = sobj.nextInt();

        MyArray Mobj1 = new MyArray(iValue1);
        Mobj1.Accept();


        System.out.println("Enter Size of Second Array : ");
        int iValue2 = sobj.nextInt();

        MyArray Mobj2 = new MyArray(iValue2);
        Mobj2.Accept(); 

        System.out.println("Elements of Array are : ");

        Mobj1.Display();
        Mobj2.Display();

    }
}